core/domain/skill_services_test.py

```diff
@@ -274,6 +274,14 @@ def test_delete_skill(self):
             skill_services.get_skill_summary_by_id(
                 self.SKILL_ID, strict=False), None)
 
+    def test_get_multi_skill_rights(self):
```

Comment:
Test the case that I mentioned above? (i.e. skill rights for skills which don't exist)

Reply:
Done.